components/Modal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Color } from "types/tailwindcss";
+
+interface IModalProps {
+  callback(action: string): void;
+  title: string;
+  message: string;
+  actions: {
+    color: Color;
+    action: string;
```

Comment:
```suggestion
    id: string;
```

[불필요한 문맥은 추가하지 마세요 - 타입스크립트를 위한 클린코드](https://github.com/738/clean-code-typescript#%EB%B6%88%ED%95%84%EC%9A%94%ED%95%9C-%EB%AC%B8%EB%A7%A5%EC%9D%80-%EC%B6%94%EA%B0%80%ED%95%98%EC%A7%80-%EB%A7%88%EC%84%B8%EC%9A%94)